docs/api.rst

```diff
@@ -0,0 +1,72 @@
+:og:description: Comprehensive API documentation for Felis.
+
+########################
+Python API Documentation
+########################
+
+.. contents:: Table of Contents
+    :local:
```

Comment:
This is unnecessary since the theme already provides a table of contents for the page.